static/index.html

```diff
@@ -1,22 +1,57 @@
-<!DOCTYPE html>
+<!doctype html>
```

Comment:
Why did you change the capitalization here?

Reply:
I formatted the file with `prettier`, that seems to be its default.

Reply:
🤔